DOCUMENTATION.md

```diff
@@ -42,6 +42,13 @@ Create an application. This function only works if you log in using credentials
 
 #### Raises:
     InvalidDeviceType: if device type is not supported.
+
+#### Examples:
+```python
+>>> resin.models.application.create('Edison','Intel Edison')
+Return: dict contains details of new application in the following format.
```

Comment:
This applies to everywhere too -- I see you added similar comments in several places.